Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM pandoc/latex:2.9.2
+LABEL dockerimage=cv-boilerplate
+
+RUN apk add -U make
+
+# Google fonts
+RUN wget --progress=bar:force:noscroll https://github.com/google/fonts/archive/main.tar.gz -O gf.tar.gz
```

Comment:
this downloads the entire collection of Google Fonts into the container 🙈 so one can choose the font they like on https://fonts.google.com/ and then use it right away